recipes/mne-realtime/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "mne-realtime" %}
+{% set version = "0.1.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: c9be35f26695bbfeb79656ead7ada2ad02175212b849479653184f29d0d594a2
+
+build:
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - python >=3.7
+    - pip
+  run:
+    - python >=3.7
+    - numpy
+    - scipy
+    - mne-base
+
+test:
+  imports:
+    - mne_realtime
+  requires:
+    - pip
+  commands:
+    - pip check
+
+about:
+  home: https://mne.tools/mne-realtime
+  license: BSD-3-Clause
+  license_family: BSD
+  license_file: LICENSE.txt
```

Comment:
This path would now need to point to the path of the new license as well, otherwise conda-build doesn't copy it over.

You can just add it as a list.

```yaml
license_file: 
  - first_license
  - Second_license
```

Reply:
@BastianZim Done!